Employment Policies/Working Remotely.md

```diff
@@ -1,11 +1,91 @@
+* [Communication and Transparency](https://github.com/clef/handbook/blob/master/Onboarding%20Documents/Communication%20and%20Transparency.md)
+
+### Extended Remote Work
+
+#### Extended remote work 
+
+***An extended remote work period includes anything longer than 2 days or any period of time where we work from a irregular place.***
+
+If you're planning to work remotely for more than two consecutive days, or you're planning to work remotely from a place that's abnormal (i.e. from a partner's home in a different city), you'll need to follow these additional guidelines. Even more than standard remote work, working remotely for an extended period of time is a privilege — it **will** require a large amount of extra work from you to make sure that the team is effective with you working remotely.
+
+Get Approval From your manager
+
+You need your manager's approval before you can work remotely for an extended period of time. This check-in should happen at least two weeks in advance, before you make any travel arrangements, and you should come prepared with a plan for how you will handle all the edge cases involved in extended remote work. In order for an employee to work remotely for an extended period of time, they should have demonstrated in the past that they are effective working remotely and upholding their quality of work. If that's not the case, the manager can and should veto the option and work with the employee to come up with a plan that would get them on track to work remotely in a more effective way.
+
+#### Give the team heads up
+
+You should give everyone on the team as much notice as possible, but at least 7 days. This will ensure everyone can plan their meetings and work accordingly. 
+
+Always take the following steps to let the team know:
+
+* Post in #office with a @channel tag so everyone sees the notification
+* Communicate verbally with the people you work closest with
+* Update your calendar to mark which days you are working remotely
+
+#### Plan & Prepare Beforehand
+
+It's your responsibility to both make sure you are effective and don't let your teammates down — regardless of the location you work from. This means that you should plan & prepare in your free time before you leave to work remotely. 
+
+A non-exhaustive list of things to ensure are in order are:
+
+* You will have a fast, consistent wi-fi connection
+* You will have a distraction-free environment to work in
+* You will have a quiet, private place to take phone calls and meetings
+* You will be able to work a full workday every day you're working remotely
+
+#### Co-working Space Subsidies
+
+Working remotely can add a financial burden on an individual. At the same time, it is also a privilege. While Clef is on its seed extension, Clef will subsidize co-working space costs for up to $10/day. Once Clef reaches profitability or raises its next major fundraising round, we will reconsider this subsidy.
```

Comment:
I would say "revisit" instead of "reconsider" because reconsider sounds like we'll take the subsidy away